### PR TITLE
new role: poetry

### PR DESCRIPTION
**Pull request by SytseGroenwold**
Creating a seperate branch and PR for the new poetry role.

### CODE REVIEW

**Comment by SytseGroenwold** [comment]
@tsmeele Please first merge the pip branch before doing this one. It will create a merge conflict, which we can address together as well for practice.

File: docs/roles/poetry.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+This role will install poetry through the official documentation's [recommended installed](https://python-poetry.org/docs/master/#installing-with-the-official-installer).
+The installtation is done on a per-user basis, as per the application's designed usage.
+The python virtualenv dependency is first installed through Ansible's `apt` module.
+This role was seperated from its playbook to facilitate reusability.
+
+## Variables
+None.
+
+## See also
+[pyenv playbook](../playbooks/poetry.md)
```
Comment (SytseGroenwold) [inline docs/roles/poetry.md:20]:
```suggestion
```

**Comment by SytseGroenwold** [comment]
Instead work with installation through runonce. Think of a runonce role which can trigger the runonce at creation time, not login.

File: docs/roles/poetry.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+## Summary
+Poetry is a Python package version management tool for easy packageing and dependency management.
+
+## Requires
+* Unix-based system.
+* Python 3.7+
+
+## Description
+This role will install poetry through the official documentation's [recommended installed](https://python-poetry.org/docs/master/#installing-with-the-official-installer).
+The installtation is done on a per-user basis, as per the application's designed usage.
```
Comment (tsmeele) [inline docs/roles/poetry.md:13]:
```suggestion
The installation is done on a per-user basis, as per the application's designed usage.
```

File: roles/poetry/defaults/main.yml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+---
+per_user: true
```
Comment (tsmeele) [inline roles/poetry/defaults/main.yml:2]: prefix with role name

**Pull request merged**